src/som/som_foodinsec_trigger.py

```diff
@@ -57,14 +57,18 @@
     parameters[f"path_admin{admin_level}_shp"],
 )
 
+output_dir = os.path.join(country_data_processed_dir, "trigger_metrics")
+output_path = os.path.join(
+    output_dir, f"{iso3}_foodinsec_trigger{suffix_output}.csv"
+)
+
 
 df_fn = pd.read_csv(os.path.join(fewsnet_dir, fewsnet_filename))
```

Comment:
Not directly part of your code refactoring above, but `fewsnet_filename` is no longer valid. They are named `somalia_...` in the file drive now. I didn't want to change in case used elsewhere, but flagging here.

Reply:
thanks for noticing, changed the names on the drive. They are referenced to in a few notebooks but the change should be either automatically reflected due to the use of the config file, or they were already out of date so won't change those now